src/api/getSecurityQuestion.js

```diff
@@ -14,20 +13,14 @@ router.post("/getSecurityQuestion", check("email").isEmail(), async function(
       errors: errors.array({ onlyFirstError: true })
     });
   }
-
-  const user = await User.findOne({ email: req.body.email });
-  if (!user) {
-    sendResponse(res, 400, "User does not exist in the database");
-    return;
-  }
-  if (!user.question) {
-    sendResponse(res, 400, "No security question set");
-    return;
+  const questionsResponse = await getSecurityQuestions();
```

Comment:
Get security questions, to my understanding, is called on the user profile page when the user is setting the security question. In this case, it should be using the token to verify the user.

Reply:
That's true, I Can do that.